src/runner/clock_test.py

```diff
@@ -0,0 +1,22 @@
+from src.runner.testing_instance import TInstance
+
+class ClockTest(TInstance):
+    def run_iteration(self):
```

Comment:
can we rename the run_iteration method to `cycle`? I think this is the industry standard term